### ISSUE TITLE
GUI Customization - change color of job titles

### ISSUE THREAD

**Issue by sys-ops** [rundeck/rundeck#4099]
**Is your feature request related to a problem? Please describe.**
Yes. It is hard to read job titles with light blue color and white background.

**Describe the solution you'd like**
I would like to be able to tweak the color of job titles through rundeck-config.properties file.

**Describe alternatives you've considered**
None.

**Additional context**
![image](https://user-images.githubusercontent.com/5460298/46671854-2e98b600-cbd6-11e8-8706-d161cdc00703.png)

**Comment by nhashem** [comment]
I would also like this enhancement.

**Comment by ahonor** [comment]
@nhashem for the same reason as @sys-ops (not enough contrast)?

**Comment by nhashem** [comment]
Yes.  The countdown timer to the next scheduled job run also used to be in green, and it is now much harder to see which jobs are scheduled to run soon.

**Comment by fdevans** [comment]
Thanks for the report. As part of a cleanup of our issue tracker, we're closing issues older than 12 months, in line with Rundeck's support policy of one year from a release's date (see the [release calendar](https://docs.rundeck.com/docs/history/release-calendar.html) for currently supported versions). This isn't a judgment that the report is invalid — many of these are likely still real — but we can't verify against a version this old.

If you're still seeing this on a currently supported release (6.1.x or 5.20.x+), please open a new issue using our [bug report template](https://github.com/rundeck/rundeck/issues/new?template=bug_report.yml) with your version and reproduction steps. Please don't reopen this issue — a fresh issue keeps things tracked against a supported version and gets it in front of us properly. We're actively working on improving how issues get submitted and triaged going forward.

Thanks for your patience,
The Rundeck Team
